lms/templates/courseware/accordion.html

```diff
@@ -6,73 +6,42 @@
 %>
 
 <%def name="make_chapter(chapter)">
-  <div class="chapter">
-      <%
-          if chapter.get('active'):
-              aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
-              active_class = ' class="active"'
-          else:
-              aria_label = chapter['display_name']
-              active_class = ''
-      %>
-      <h3 ${active_class} aria-label="${aria_label}">
-        <a href="#">
-          ${chapter['display_name']}
+<%
+if chapter.get('active'):
+    aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
+    active_class = 'active"'
+else:
+    aria_label = chapter['display_name']
+    active_class = ''
+%>
+<button class="button-chapter chapter ${active_class}" id="${chapter['display_id']}-parent" aria-controls="${chapter['display_id']}-child" aria-expanded="false">
+    <span class="group-heading ${active_class}" aria-label="${aria_label}">
+        <span class="icon fa fa-caret-right" aria-hidden="true"></span>
+        ${chapter['display_name']}
+    </span>
+</button>
+<div class="chapter-content-container" id="${chapter['display_id']}-child" tabindex="-1" aria-expanded="false" role="region" aria-labelledby="${chapter['display_id']}-parent" aria-hidden="false">
```

Comment:
@clrux I think I want to use role="group" here. I think its more semantic than region, which implies a much larger area of items that may or may not be related.  This is clearly a group here, a group of links.  On that note, I think we need to make the label more precise. Currently, it doesn't inform the user that its a submenu.  Instead of an aria-labelledby here, can we make it an aria-label that reports the parent name and appends "submenu" so that it gets announced as "Introduction submenu, group"